include/exadg/incompressible_navier_stokes/user_interface/parameters.cpp

```diff
@@ -549,6 +540,7 @@ Parameters::check(dealii::ConditionalOStream const & pcout) const
   }
 
   // VARIABLE VISCOSITY MODELS
+  // TODO
   if(viscosity_is_variable() and
      temporal_discretization == TemporalDiscretization::BDFDualSplittingScheme)
```

Comment:
I think I added this TODO because the code should now be able to treat the variable viscosity implicitly for the dual splitting scheme, similarly to the pressure correction scheme. When should we fix this? E.g. do you want to test this once the present PR is merged and then you/I remove this if-statement?